crates/turborepo-lib/src/run/builder.rs

```diff
@@ -108,9 +118,21 @@ impl RunBuilder {
             ui,
             version,
             experimental_ui,
+            entrypoint_tasks: None,
+            should_print_prelude_override: None,
         })
     }
 
+    pub fn with_entrypoint_tasks(mut self, entrypoint_tasks: Vec<TaskId<'static>>) -> Self {
+        self.entrypoint_tasks = Some(entrypoint_tasks);
+        self
+    }
+
+    pub fn hide_prelude(mut self) -> Self {
```

Comment:
I added this because the prelude makes less sense with re-run tasks. It prints the packages in scope, but the tasks in those packages are not always run, since the task graph is pruned.